src/core/directives/google-map-polyline.ts

```diff
@@ -0,0 +1,224 @@
+import {AfterContentInit, ContentChildren, Directive, EventEmitter, OnChanges, OnDestroy, QueryList, SimpleChanges} from '@angular/core';
+import {Subscription} from 'rxjs/Rx';
+
+import {PolyMouseEvent} from '../services/google-maps-types';
+import {PolylineManager} from '../services/managers/polyline-manager';
+
+import {SebmGoogleMapPolylinePoint} from './google-map-polyline-point';
+
+let polylineId = 0;
+
+@Directive({
+  selector: 'sebm-google-map-polyline',
+  inputs: [
+    'clickable', 'draggable', 'editable', 'geodesic', 'strokeColor', 'strokeWeight',
```

Comment:
please change the input name from draggable to polylineDraggable (see here why: https://github.com/SebastianM/angular2-google-maps/pull/556#discussion_r74542721)